idb/dummy.go

```diff
@@ -218,7 +218,7 @@ type IndexerDb interface {
 	// YieldTxns returns a channel that produces the whole transaction stream after some round forward
 	YieldTxns(ctx context.Context, prevRound int64) <-chan TxnRow
 
-	CommitRoundAccounting(updates RoundUpdates, round, rewardsBase uint64) (err error)
+	CommitRoundAccounting(updates RoundUpdates, round uint64, blockPtr *types.Block) (err error)
```

Comment:
What do you think about adding `round` to the `RoundUpdates` object instead of adding a new parameter?

Reply:
We could put the block data in there as well actually.

Reply:
Yes, and I got started making this change, then stopped. RoundUpdates is used heavily in accounting/ code; but those two fields are just metadata to CommitRoundAccounting().
Alternate theory: RoundUpdates struct definition could move to the accounting package.
Not sure which way I want to go so I guess leave it for now?